site/server/test/Database.scala

```diff
@@ -0,0 +1,32 @@
+package test.database
+
+import java.io.File
+import javax.sql.DataSource
+import play.api.db.evolutions._
+import play.api.db.{ Database, Databases }
+import doobie.imports._
+import scalaz.concurrent.Task
+
+object TestDatabase {
+  val testDriver = "org.h2.Driver"
+  def testUrl = {
+    val dbFile = File.createTempFile("temporary-db", ".tmp").getAbsolutePath()
+    s"jdbc:h2:file:$dbFile;DATABASE_TO_UPPER=false"
+  }
+  val testUsername = "sa"
+  val testPassword = ""
+
+  def db: Database = {
```

Comment:
The name `db` it's misleading since it applies effects in applyEvolutions. Perhaps something like `updateDB()`